stanford/bin/dump-logs

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+rsync_sudo() {
+    rsync --rsync-path='sudo rsync' -avh --progress ${@}
+}
+rsync_logs() {
+    machine=${1}
+    destination=${2}
+    shift; shift;
+    rsync_sudo ${@} "${machine}:/var/log/" "${destination}"
+    rsync_sudo ${@} "${machine}:/edx/var/log/" "${destination}/edx"
+}
+destination_root=/edx/var/log
+deployments="cme"
+clusters="analytics-api build certs coursegraph deploy director edxapp esforum forum insights jump maintenance nat notifier rabbitmq scheduler worker xqueue"
+for deployment in ${deployments}; do
+    for cluster in ${clusters}; do
+        for number in {0..4}; do
+            box="${cluster}${number}"
+            machine=${box}.${deployment}.vpc
+            if ssh -q -o ConnectTimeout=1 "${machine}" 'exit' 2>/dev/null; then
```

Comment:
Check and see if we can access the host via SSH.
It tries to connect and then immediately exit, else it times out after 1second.
If we can connect, we assume it's a real/existing host and we should try copying logs.